client/hosting/overview/components/migration-overview/components/migration-pending/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { LoadingPlaceholder } from '@automattic/components';
+import { Button } from '@wordpress/components';
+import { translate } from 'i18n-calypso';
+import { useCallback, useEffect } from 'react';
+import { HostingHeroButton } from 'calypso/components/hosting-hero';
+import { useMigrationCancellation } from 'calypso/data/site-migration/landing/use-migration-cancellation';
+import { addQueryArgs } from 'calypso/lib/url';
+import { getMigrationType } from 'calypso/sites-dashboard/utils';
+import { useDispatch } from 'calypso/state';
+import { requestSite } from 'calypso/state/sites/actions';
+import Cards from '../cards';
```

Comment:
Nit: I feel this is too generic and would be hard to maintain, I'd suggest relating it with the actual component name, something like MigrationOverviewCards.